src/java/org/apache/cassandra/schema/SchemaKeyspace.java

```diff
@@ -70,41 +72,14 @@ private SchemaKeyspace()
     private static final boolean FLUSH_SCHEMA_TABLES = Boolean.parseBoolean(System.getProperty("cassandra.test.flush_local_schema_changes", "true"));
     private static final boolean IGNORE_CORRUPTED_SCHEMA_TABLES = Boolean.parseBoolean(System.getProperty("cassandra.ignore_corrupted_schema_tables", "false"));
 
-    public static final String KEYSPACES = "keyspaces";
-    public static final String TABLES = "tables";
-    public static final String COLUMNS = "columns";
-    public static final String DROPPED_COLUMNS = "dropped_columns";
-    public static final String TRIGGERS = "triggers";
-    public static final String VIEWS = "views";
-    public static final String TYPES = "types";
-    public static final String FUNCTIONS = "functions";
-    public static final String AGGREGATES = "aggregates";
-    public static final String INDEXES = "indexes";
-
-    /**
-     * The order in this list matters.
-     *
-     * When flushing schema tables, we want to flush them in a way that mitigates the effects of an abrupt shutdown whilst
-     * the tables are being flushed. On startup, we load the schema from disk before replaying the CL, so we need to
-     * try to avoid problems like reading a table without columns or types, for example. So columns and types should be
-     * flushed before tables, which should be flushed before keyspaces.
-     *
-     * When truncating, the order should be reversed. For immutable lists this is an efficient operation that simply
-     * iterates in reverse order.
-     *
-     * See CASSANDRA-12213 for more details.
-     */
-    public static final ImmutableList<String> ALL =
-        ImmutableList.of(COLUMNS, DROPPED_COLUMNS, TRIGGERS, TYPES, FUNCTIONS, AGGREGATES, INDEXES, TABLES, VIEWS, KEYSPACES);
-
```

Comment:
It feels like moving the table names out of `SchemaKeyspace` really fragments things (given `Schema` is moving toward being an abstraction on top of `SchemaKeyspace`, not the other way around).

Reply:
(...and of course this would greatly reduce the size of the diff/patch)

Reply:
On the other hand, making `SchemaKeyspace` package private, so nobody calls the non-synched methods directly also has value. I have mixed feelings here, I'll give it another thought.

Reply:
Actually we can extract just the cts to a public class and get the best of both worlds right?

Reply:
If you move the cts to the `SchemaKeyspace` class then you can't make that class package private. Which imo is good at preventing anybody adding a `public` method to `SchemaKeyspace`. That reinforces `Schema` to be the public API as it's the public class.

But I don't have a strong opinion here. @smiklosovic since you dropped a comment feel free to untie us lol! wdyt?

Reply:
I intend to do the rest of PRs tomorrow + rebase etc Unless Stefan unties us I will let the cts in a separate class to favor the package private approach unless you throw at me a hard -1 :-) I am 60% 40% in favor of the package private option.

Reply:
Made a couple naming comments on the constants class, but I'm okay w/ that approach. I'm also good at this point w/ everything else outside the test, but let me comment on that separately.

Reply:
Happy to wait then :-)